setup.py

```diff
@@ -62,27 +77,116 @@ def package_vars(version_file):
+    install_requires=[
+        "2captcha-python>=1.0.3",
+        "appdirs>=1.4.4",
+        "appnope>=0.1.0",
+        "attrs>=20.3.0",
+        "backcall>=0.2.0",
+        "bcrypt>=3.2.0",
+        "beautifulsoup4>=4.9.1",
+        "boto3>=1.14.39",
+        "botocore>=1.17.63",
+        "certifi>=2020.12.5",
+        "cffi>=1.14.4",
+        "cfgv>=3.2.0",
+        "chardet>=3.0.4",
+        "click>=7.1.2",
+        "cognitojwt>=1.2.2",
+        "cryptography>=3.3.2",
+        "decorator>=4.4.2",
+        "distlib>=0.3.1",
+        "dnspython>=2.1.0",
+        "docutils>=0.15.2",
+        "ecdsa>=0.14.1",
+        "Faker>=5.8.0",
+        "filelock>=3.0.12",
+        "Flask>=1.1.2",
+        "Flask-Cors>=3.0.9",
+        "gunicorn>=20.0.4",
+        "identify>=1.5.5",
+        "idna>=2.10",
+        "iniconfig>=1.1.1",
+        "ipdb>=0.13.4",
+        "ipython>=7.18.1",
+        "ipython-genutils>=0.2.0",
+        "itsdangerous>=1.1.0",
+        "jedi>=0.17.2",
+        "Jinja2>=2.11.3",
+        "jmespath>=0.10.0",
+        "MarkupSafe>=1.1.1",
+        "marshmallow>=3.7.1",
+        "mccabe>=0.6.1",
+        "mongomock>=3.22.0",
+        "mypy-extensions>=0.4.3",
+        "nodeenv>=1.5.0",
+        "packaging>=20.9",
+        "paramiko>=2.7.2",
+        "parso>=0.7.1",
+        "pathspec>=0.8.0",
+        "pexpect>=4.8.0",
+        "pickleshare>=0.7.5",
+        "pluggy>=0.13.1",
+        "prompt-toolkit>=3.0.8",
+        "ptyprocess>=0.6.0",
+        "py>=1.10.0",
+        "pyasn1>=0.4.8",
+        "pycodestyle>=2.6.0",
+        "pycparser>=2.20",
+        "pyflakes>=2.2.0",
+        "Pygments>=2.7.4",
+        "pymongo>=3.11.1",
+        "PyNaCl>=1.4.0",
+        "pyparsing>=2.4.7",
+        "python-dateutil>=2.8.1",
+        "python-dotenv>=0.14.0",
+        "python-jose>=3.2.0",
+        "PyYAML>=5.4",
+        "regex>=2020.10.11",
+        "requests>=2.24.0",
+        "rsa>=4.7",
+        "s3transfer>=0.3.4",
+        "selenium>=3.141.0",
+        "sentinels>=1.0.0",
+        "six>=1.15.0",
+        "soupsieve>=2.1",
+        "sshtunnel>=0.3.1",
+        "text-unidecode>=1.3",
+        "toml>=0.10.1",
+        "traitlets>=5.0.5",
+        "TwoCaptcha>=0.0.1",
+        "typed-ast>=1.4.1",
+        "typing-extensions>=3.7.4.3",
+        "undetected-chromedriver>=2.1.2",
+        "urllib3>=1.25.11",
+        "validators>=0.18.2",
+        "wcwidth>=0.2.5",
+        "Werkzeug>=1.0.1",
+    ],
     extras_require={
         "test": [
-            "pre-commit",
-            # coveralls 1.11.0 added a service number for calls from
-            # GitHub Actions. This caused a regression which resulted in a 422
-            # response from the coveralls API with the message:
-            # Unprocessable Entity for url: https://coveralls.io/api/v1/jobs
-            # 1.11.1 fixed this issue, but to ensure expected behavior we'll pin
-            # to never grab the regression version.
-            "coveralls != 1.11.0",
```

Comment:
Why did you remove `coveralls` here?

Reply:
My bad. I'll add it back in. I just put everything from the requirements-test.txt file into that section.

Reply:
Added back in https://github.com/cisagov/domain-manager-api/pull/194/commits/bb0d690ef316e5c0519d27ead1aa253d4021537f

Reply:
Excellent, thanks.